src/m365/planner/Task.ts

```diff
@@ -7,3 +7,7 @@ export interface Task {
   planId?: string;
   bucketId?: string;
 }
+
+export interface BetaTask extends Task {
```

Comment:
Let's extend `PlannerTask` from Graph typings instead of our custom `Task` type that we'll decommission in the future

Reply:
Will do it